worker/codalabworker/aws_batch.py

```diff
@@ -19,6 +20,9 @@ def current_time():
 
 BYTES_PER_MEGABYTE = 1024 * 1024
 
+# AWS Batch always requires the amount of memory to be specified. We provide a reasonable default here.
+BATCH_DEFAULT_MEMORY = 100 * BYTES_PER_MEGABYTE
```

Comment:
Perhaps this should be higher (e.g., 1GB), so that you can easily run one-offs like `cl run <simple command>` without worrying too much...